tests/Propel/Tests/Generator/Platform/PlatformMigrationTestProvider.php

```diff
@@ -198,6 +198,7 @@ public function providerForTestGetModifyTableIndicesDDL()
     <table name="foo">
         <column name="id" primaryKey="true" type="INTEGER" autoIncrement="true" />
         <column name="bar" type="INTEGER" />
+        <column name="bax" type="VARCHAR" size="12" required="true" />
```

Comment:
Usually, these names should be more speaking but that would be too much this PR. Can you please follow foo, bar, baz, baz naming here, and move the new column after the `baz` column?